dev/CommonStyles/TestUI/MenuFlyoutPage.xaml

```diff
@@ -10,36 +10,62 @@
     mc:Ignorable="d">
 
     <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Margin="12">
+        <Grid.ColumnDefinitions>
```

Comment:
You can just have a grid with two columns and two rows and put textblocks in there accordingly, instead of two extra stack panels